crates/accelerate/Cargo.toml

```diff
@@ -29,7 +29,7 @@ version = "^0.15.6"
 features = ["rayon"]
 
 [dependencies.hashbrown]
-version = "0.12.3"
+version = "0.13.2"
```

Comment:
With #10541 merged now we can bump the hashbrown version to 0.14.0 because our msrv is 1.64.
```suggestion
version = "0.14.0"
```

Reply:
these would need to come along with updates to the lock file as well, so probably better not to do inline?